tests/phpunit/elementor/app/import-export/processes/test-revert.php

```diff
@@ -73,6 +73,8 @@ public function test_run__revert_all_one_imported_only() {
 		$this->assertEquals( $base_posts, $after_revert__posts );
 		$this->assertNotEquals( $after_import__posts, $after_revert__posts );
 
+		$this->assert_valid_revert_sessions_option();
```

Comment:
Maybe pass the sessions as a parameter, because this way it feels obscure

Reply:
Done